### ISSUE TITLE
Peer Review

### ISSUE THREAD

**Issue by ian-shryock** [amynwarnock/edld652-finalproject#10]
This is such an extensive project, excellent work! I really like the development of each visualization and the way you approached the data from a number of angles. Honestly, it's hard to choose among your many plots because they each have their own strengths. In general, I think that some of the plots/choices between the options depends on what you want the takeaway to be. Once that's clear, adding in titles that convey that message would be really helpful, especially for the more complicated plots (the plot of special ed spending positively impacting outcomes does a good job of this). Overall, it seems like there are tons of options you can choose from for the final project and lots of different routes you can take going forward. 

@datalorax

**Comment by datalorax** [comment]
Thanks Ian - I really appreciate these comments and your careful attention to their project. I was hoping that you would actually embed comments and suggest code edits to their actual Rmd, which could then be shared via a pull request. In lieu of that, this is a good option.

**Comment by ian-shryock** [comment]
Oh that's strange....I did leave comments within the actual file...I thought I'd pushed those changes but maybe not. Just re-pushed them (I'll go check the other reviews I did)

**Comment by datalorax** [comment]
Yeah in this case it looks like you just filed an issue rather than submitting a pull request. The process should be

1. Fork their repo
2. Clone your fork (from your account)
3. Make edits/suggestions
4. Push changes to your fork
5. Come back to this repo, select Pull Requests -> Open new request -> Compare across forks
6. Submit pull request

Hope this helps, but happy to chat if you'd like.
